tokio-util/src/codec/bytes_codec.rs

```diff
@@ -6,12 +6,12 @@ use std::io;
 
 /// A simple `Codec` implementation that just ships bytes around.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Default)]
-pub struct BytesCodec(());
+pub struct BytesCodec;
```

Comment:
This is added to prevent pattern matching. If `(())` is removed, we are unable to add fields in the future.

Reply:
Fixed, other structs with the same definition aren't exported from crate.